tests/components/aquacell/conftest.py

```diff
@@ -28,7 +28,7 @@ def mock_setup_entry() -> Generator[AsyncMock, None, None]:
 
 
 @pytest.fixture
-def mock_aquacell_api() -> Generator[AsyncMock, None, None]:
+def mock_aquacell_api() -> Generator[AsyncMock]:
```

Comment:
Same here.
```suggestion
def mock_aquacell_api() -> Generator[Mock]:
```

For this one, it would be really good to use `Generator[MockOf[AquacellApi]]` but it's not available yet.